db/migrate/20161110203212_create_membership_applications.rb

```diff
@@ -1,4 +1,4 @@
-class CreateMembershipApplications < ActiveRecord::Migration[5.0]
+class CreateMembershipApplications < ActiveRecord::Migration[5.1]
```

Comment:
Just curious:  do you *have to* update the version?  My understanding is that the version is there so that we understand the syntax/limitations of the version when the migration was written.  But I could surely be wrong about that.  

If I'm correct, then we should not change the version for the migrations.  However, I don't think we have any migrations where that really matters, so I'm fine with the change regardless.